deploy/katello.spec

```diff
@@ -138,7 +151,7 @@ usermod -a -G katello-shared tomcat
 %dir %{homedir}
 %config(missingok) %{_sysconfdir}/cron.weekly/katello-remove-orphans
 
-%pre
+%pre common
```

Comment:
I think the stuff below here can be removed since we don't use the katello or katello-shared user/groups anymore. Not that you need to do that here, but an issue to track would be nice.